### PR TITLE
Implementing rhods temmplate

### PR DESCRIPTION
**Pull request by lucferbux**
---

name: Pull Request
about: Contribute a new pull request
title: 'Add RHODS e-mail Template'

---

**Add RHODS Email Template**
- 
- Added Instant Email for the Red Hat OpenShift Data Science platform

**Closing issues**
- [JIRA RHODS-852](https://issues.redhat.com/browse/RHODS-852)

**Additional context**
- I'm creating a Draft Pull Request as we are not sure of the content of the email, we will use this PR to discuss the feature.

### CODE REVIEW

**Review by lucferbux — COMMENTED** [review]
@maulikjs here it is the review of the templating

File: src/main/resources/templates/Rhods/insightsEmailBody.html [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+
+                    <!-- Head -->
+                    <tr>
+                        <td class="rh-head">
+                            <table class="rh-masthead" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <!--Red Hat logo-->
+                                    <td align="center" class="rh-masthead__brand" bgcolor="#151515">
+                                        <a href="https://cloud.redhat.com/" target="_blank" class="rh-masthead__brand-link">
+                                            <img src="https://cloud.redhat.com/apps/application-services/485bae8d.svg" alt="Red Hat logo" width="182" height="90" class="rh-masthead__brand-img" />
```
[the remark this answers is not in the document]
Comment (maulikjs) [reply src/main/resources/templates/Rhods/insightsEmailBody.html:457]: Hey Lucas this should be fine for now thanks!

**Comment by gwenneg** [comment]
We recently changed the Maven project structure so I rebased and updated your PR. It is now compatible with the new structure.

**Comment by lucferbux** [comment]
> We recently changed the Maven project structure so I rebased and updated your PR. It is now compatible with the new structure.

Thank you very much @gwenneg

**Comment by lucferbux** [comment]
I think we are not gonna use this solution, although I merged the main branch into these changes, I will close this PR if we manage to implement the alternate solution.

**Pull request closed without merging**